packages/pyxis-react/src/stories/components/Counter/Counter.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import { Meta, Story } from '@storybook/react';
+
+import React from 'react';
+import Counter, { CounterProps } from './Counter';
+
+export default {
+  title: 'Components - Form/Counter/All Stories',
```

Comment:
When the React component is not "final", we add under-construction emoji:  🚧
```suggestion
  title: 'Components - Form/Counter  🚧/All Stories',
```
Could you change it even to other files?